table/block.h

```diff
@@ -283,16 +284,10 @@ class BlockIter final : public InternalIterator {
   virtual Status status() const override { return status_; }
   virtual Slice key() const override {
     assert(Valid());
-    return key_includes_seq_ ? key_.GetInternalKey() : key_.GetUserKey();
+    return key_.GetInternalKey();
   }
```

Comment:
Can we leave the function not implemented?

Reply:
Meta blocks could still legitimately use BlockIter class.